bot.py

```diff
@@ -121,10 +155,42 @@ async def my_task(self):
         thread = await message.create_thread(name=date_string, auto_archive_duration=1440)
         for react in EMOJI_MAP.keys():
             await message.add_reaction(react)
-
+    
+    @tasks.loop(hours=1)
+    async def adjust_probs(self):
+        for author in dt.AUTHOR_DICT:
+            dt.AUTHOR_DICT[author] = max(0, dt.AUTHOR_DICT[author] - adjust_prob_by)
 
 intents = discord.Intents.default()
 intents.message_content = True
 
+bot = commands.Bot(command_prefix="/", intents=intents)
+
 client = MyClient(intents=intents)
+
+@client.tree.command()
+async def new_response(interaction: discord.Interaction, response: str, condition: str):
+    """Adds a new message, and a Boolean condition that causes it to be posted"""
+    global tree
+    
+    try:
+        parsed_condition = parse_string(condition)
+    
+    except Exception as e:
+        await interaction.response.send_message("Couldn't parse condition: " + str(e))
+        return
```

Comment:
give an example of a valid input, otherwise we will never figure it out. Ideally we have the boy show an example in the `/thing AND(x.y)` but that's hard ill settle for this.